source/server/management/exploit-patches.rst

```diff
@@ -0,0 +1,30 @@
+===============
+Exploit Patches
+===============
+
+In recent Sponge builds (SpongeForge 974+), SpongeForge and SpongeVanilla patch a few client-server exploits. Whenever
+the implementations detect a user performing an exploit, they are kicked from the server with a message explaining why
+they were kicked. If enabled, a log message is also sent to the console. More exploit patches may be added in the
+future.
+
+Exploits Patched by Sponge Implementations
+==========================================
+
+#. Sign command exploit where a client could run a command such as 'op'
```

Comment:
Might want to include the conf entry here (and below), like `logging.exploit-sign-command-updates`, for quick and easy reference.